server/src/metadata/object-metadata/hooks/before-create-one-object.hook.ts

```diff
@@ -1,11 +1,26 @@
-import { Injectable, UnauthorizedException } from '@nestjs/common';
+import {
+  ForbiddenException,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
 
 import {
   BeforeCreateOneHook,
   CreateOneInputType,
 } from '@ptc-org/nestjs-query-graphql';
 
 import { CreateObjectInput } from 'src/metadata/object-metadata/dtos/create-object.input';
+import { standardObjectsNames } from 'src/workspace/workspace-manager/standard-objects/standard-object-metadata';
+
+const OBJECT_TYPES = [
+  'featureFlag',
+  'refreshToken',
+  'UserWorkspaceMemberName',
```

Comment:
I'm not sure about those, the only core tables we have today are:
- featureFlag,
- refreshToken
- user
- workspace